Raven.Voron/Voron/Trees/Page.cs

```diff
@@ -172,6 +172,19 @@ public ushort PageSize
             }
         }
 
+        public List<long> GetAllOverflowPages()
+        {
+            var results = new List<long>(NumberOfEntries);
```

Comment:
This is VERY wrong.
You are getting entries that are NOT overflow entries, so you are effectively trying to load random memory entries.